src/infra/account/accountCmdRepo.go

```diff
@@ -0,0 +1,180 @@
+package accountInfra
+
+import (
+	"errors"
+	"os"
+	"os/user"
+	"time"
+
+	"github.com/goinfinite/os/src/domain/dto"
+	"github.com/goinfinite/os/src/domain/entity"
+	"github.com/goinfinite/os/src/domain/valueObject"
+	infraHelper "github.com/goinfinite/os/src/infra/helper"
+	internalDbInfra "github.com/goinfinite/os/src/infra/internalDatabase"
+	dbModel "github.com/goinfinite/os/src/infra/internalDatabase/model"
+	"github.com/google/uuid"
+	"golang.org/x/crypto/bcrypt"
+)
+
+type AccountCmdRepo struct {
+	persistentDbSvc *internalDbInfra.PersistentDatabaseService
+}
+
+func NewAccountCmdRepo(
+	persistentDbSvc *internalDbInfra.PersistentDatabaseService,
+) *AccountCmdRepo {
+	return &AccountCmdRepo{
+		persistentDbSvc: persistentDbSvc,
+	}
+}
+
+func (repo *AccountCmdRepo) Create(
+	createDto dto.CreateAccount,
+) (accountId valueObject.AccountId, err error) {
+	passHash, err := bcrypt.GenerateFromPassword(
+		[]byte(createDto.Password.String()), bcrypt.DefaultCost,
+	)
+	if err != nil {
+		return accountId, errors.New("PasswordHashError: " + err.Error())
+	}
+
+	usernameStr := createDto.Username.String()
+
+	_, err = infraHelper.RunCmd(
+		"useradd", "-m",
+		"-s", "/usr/sbin/nologin",
+		"-p", string(passHash),
+		usernameStr,
+	)
+	if err != nil {
+		return accountId, errors.New("UserAddFailed: " + err.Error())
+	}
+
+	userInfo, err := user.Lookup(usernameStr)
+	if err != nil {
+		return accountId, errors.New("UserLookupFailed: " + err.Error())
+	}
+
+	accountId, err = valueObject.NewAccountId(userInfo.Uid)
+	if err != nil {
+		return accountId, err
+	}
+
+	groupId, err := valueObject.NewGroupId(userInfo.Gid)
+	if err != nil {
+		return accountId, err
+	}
+
+	nowUnixTime := valueObject.NewUnixTimeNow()
+	accountEntity := entity.NewAccount(
+		accountId, groupId, createDto.Username, nowUnixTime, nowUnixTime,
+	)
+
+	accountModel, err := dbModel.Account{}.ToModel(accountEntity)
+	if err != nil {
+		return accountId, err
+	}
+
+	err = repo.persistentDbSvc.Handler.Create(&accountModel).Error
+	if err != nil {
+		return accountId, err
+	}
+
+	return accountId, nil
+}
+
+func (repo *AccountCmdRepo) getUsernameById(
+	accountId valueObject.AccountId,
+) (valueObject.Username, error) {
+	accountQuery := NewAccountQueryRepo(repo.persistentDbSvc)
+	accountEntity, err := accountQuery.ReadById(accountId)
+	if err != nil {
+		return "", err
+	}
+
+	return accountEntity.Username, nil
+}
+
+func (repo *AccountCmdRepo) Delete(accountId valueObject.AccountId) error {
+	username, err := repo.getUsernameById(accountId)
+	if err != nil {
+		return err
+	}
+
+	_, err = infraHelper.RunCmd("pgrep", "-u", accountId.String())
+	if err == nil {
+		_, _ = infraHelper.RunCmd("pkill", "-9", "-U", accountId.String())
+	}
+
+	_, err = infraHelper.RunCmd("userdel", "-r", username.String())
+	if err != nil {
+		return err
+	}
+
+	accountModel := dbModel.Account{}
+	accountIdUint64 := accountId.Uint64()
```

Comment:
This is used only once therefore this var shouldn't exist, you can just go accountId.Uint64() right on the Delete().